blazor/scheduler/accessibility.md

```diff
@@ -19,17 +19,23 @@ The following ARIA attributes are used in the Scheduler:
 
 | Property | Functionalities |
 |-------|---------|
-| role | It  gives assistive technologies about how to handle each element in a widget. |
+| role="main" | This attribute added to the Scheduler element describes the actual role of the element and denote it as a main and unique content. |
+| role="button" | This attribute is assigned to the appointments of Scheduler, to denote it as a clickable element. |
+| role="grid" | This role is added to the scheduler in content table element and describes the table that contains collection of rows. |
 | aria-disabled | It indicates the disabled state of the Scheduler. |
 | aria-selected | It indicates the currently selected cell of the Scheduler. |
 | aria-live | It indicates a string value that labels the Scheduler element. |
-| aria-label | It indicates the disabled state of the Scheduler and its items. |
+| aria-label | This attribute is assigned to the Scheduler parent element and its default value is Scheduler's current date. On every time, the date is navigated, this attribute is updated with appropriate current date values. It is also assigned to other scheduler UI elements such as previous and next date navigation buttons depicting its purpose, div element displaying date range in the header bar and appointment elements. |
+| aria-readonly | This attribute is assigned to the appointments of Scheduler and defaults to `false`, which indicates that the appointments are editable by default. When it's value changes to `true`, it denotes that the appointments are in a not editable state. |
+| aria-grabbed | This attribute is assigned to the appointments of Scheduler and defaults to `true`, which indicates that the appointments are allowed to move over the Scheduler. When it's value changes to `false`, it indicates that the drag and drop of appointments are not allowed. |
```

Comment:
remove unused attributes, recently we did some improvements in accessibility, include that changes too.